packages/design-system/src/components/tooltip/index.js

```diff
@@ -160,14 +196,15 @@ function Tooltip({
     (evt) => {
       setShown(false);
       onPointerLeave(evt);
+      resetPlacement();
```

Comment:
the other option for where to put this/structure is a use effect when shown becomes false. The reason I'd not done that before is so that it's not called on initial set up of allll the tooltips. Thoughts?

Reply:
I think this is fine. Saves us that initial reset call, and calling it here (and onBlur) seems more explicit.